packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -27,10 +27,41 @@ export interface RadioGroupState extends RadioGroupProps {
    */
   selectedButtonRef?: React.MutableRefObject<any>;
 
+  /**
+   * Populate the values arrays
```

Comment:
What are value arrays? What values are being added/removed from the Radio's? 

(clarify comments :) )

Reply:
Thanks for the suggestion! The values array is an array of all of the Radios' keys. Updated the comments with a little more description.